evil-collection-minibuffer.el

```diff
@@ -58,6 +58,12 @@ it does not have a mode."
                      minibuffer-local-isearch-map))
     (evil-define-key* 'normal map (kbd "<escape>") 'abort-recursive-edit)
     (evil-define-key* 'normal map (kbd "<return>") 'exit-minibuffer))
+
+  (with-eval-after-load 'ivy
+    (defvar ivy-minibuffer-map)
```

Comment:
I wonder if it makes sense for the evil-ivy file to take care of this part?

It chould check for the defcustom for minibuffer and enable these keys?

Reply:
That does make sense. But should it only set the bindings when `evil-collection-setup-minibuffer` is non-nil? I think they can be set regardless, since they are for normal state.

Reply:
I think so since this would have only gotten called if evil-collction-minibuffer-setup was called which makes it more obvious from the ivy file what the intention there is when trying to bind those normal state variables.

Reply:
It might be reasonable to leave it in this file too. It just jumped out when I saw the eval-after-load 'ivy.

Reply:
I'll leave it as is for now then. I'll move it later if you decide that's more appropriate. :-)

Reply:
Yeah, I think it's more appropriate in `evil-ivy` too. If we all agree, let's move it there.